lib/src/components/NxProgressBar/__tests__/NxProgressBar.test.tsx

```diff
@@ -4,191 +4,176 @@
-      function() {
-        const defaultComponent = getShallowComponent({ showCounter: true, label: 'label' });
-        const normalComponent = getShallowComponent({ variant: 'normal', showCounter: true, label: 'label' });
-        const inlineComponent = getShallowComponent({ variant: 'inline', showCounter: true, label: 'label' });
-        const smallComponent = getShallowComponent({ variant: 'small', showCounter: true, label: 'label' });
-        const fullComponent = getShallowComponent({ variant: 'full', showCounter: true, label: 'label' });
-
-        const counterSelector = '.nx-progress-bar__counter';
-        const labelTextSelector = '.nx-progress-bar__label-text';
-
-        expect(defaultComponent).toHaveClassName('nx-progress-bar--normal');
-        expect(defaultComponent).not.toHaveClassName('nx-progress-bar--undefined');
-        expect(defaultComponent.find(counterSelector)).toExist();
-        expect(defaultComponent.find(labelTextSelector)).toExist();
-
-        expect(normalComponent).toHaveClassName('nx-progress-bar--normal');
-        expect(normalComponent).not.toHaveClassName('nx-progress-bar--undefined');
-        expect(normalComponent.find(counterSelector)).toExist();
-        expect(normalComponent.find(labelTextSelector)).toExist();
-
-        expect(inlineComponent).toHaveClassName('nx-progress-bar--inline');
-        expect(inlineComponent).not.toHaveClassName('nx-progress-bar--undefined');
-        expect(inlineComponent.find(counterSelector)).not.toExist();
-        expect(inlineComponent.find(labelTextSelector)).not.toExist();
-
-        expect(smallComponent).toHaveClassName('nx-progress-bar--small');
-        expect(smallComponent).not.toHaveClassName('nx-progress-bar--undefined');
-        expect(smallComponent.find(counterSelector)).toExist();
-        expect(smallComponent.find(labelTextSelector)).not.toExist();
-
-        expect(fullComponent).toHaveClassName('nx-progress-bar--full');
-        expect(fullComponent).not.toHaveClassName('nx-progress-bar--undefined');
-        expect(fullComponent.find(counterSelector)).toExist();
-        expect(fullComponent.find(labelTextSelector)).toExist();
-      }
-  );
+  const minimalProps = {
+        value: 0,
+        label: 'current progress'
+      },
+      quickRender = rtlRender<Props>(NxProgressBar, minimalProps),
+      renderEl = rtlRenderElement<Props>(NxProgressBar, minimalProps);
+
+  it('render a <label> as the top level element', function() {
+    expect(renderEl()!.tagName).toBe('LABEL');
+  });
 
-  describe('showSteps', function() {
-    it('adds an .nx-progress-bar__step-container when true', function() {
-      expect(getMountedComponent()).not.toContainMatchingElement('.nx-progress-bar__step-container');
-      expect(getMountedComponent({ showSteps: true })).toContainMatchingElement('.nx-progress-bar__step-container');
+  it('renders a <progress> element as a child', function() {
```

Comment:
Doesn't match what' being tested
```suggestion
  it('renders an element with a role of "progressbar"', function() {
```